airbyte-integrations/bases/connector-acceptance-test/README.md

```diff
@@ -3,54 +3,86 @@ This package gathers multiple test suites to assess the sanity of any Airbyte co
 It is shipped as a [pytest](https://docs.pytest.org/en/7.1.x/) plugin and relies on pytest to discover, configure and execute tests.
 Test-specific documentation can be found [here](https://docs.airbyte.com/connector-development/testing-connectors/connector-acceptance-tests-reference/)).
 
-## Running the acceptance tests on a source connector:
-1. `cd` into your connector project (e.g. `airbyte-integrations/connectors/source-pokeapi`)
-2. Edit `acceptance-test-config.yml` according to your need. Please refer to our [Connector Acceptance Test Reference](https://docs.airbyte.com/connector-development/testing-connectors/connector-acceptance-tests-reference/) if you need details about the available options.
-3. Build the connector docker image ( e.g.: `docker build . -t airbyte/source-pokeapi:dev`)
-4. Use one of the following ways to run tests (**from your connector project directory**)
+## Configuration
+The acceptance tests are configured via the `acceptance-test-config.yml` YAML file, which is passed to the plugin via the `--acceptance-test-config` option.
+
+## Running the acceptance tests locally
+Note there are MANY ways to do this at this time, but we are working on consolidating them.
+
+Which method you choose to use depends on the context you are in.
+
+Pre-requisites:
+- Setting up a Service Account for Google Secrets Manager (GSM) access. See [here](https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connectors/ci_credentials/README.md)
+- Ensuring that you have the `GCP_GSM_CREDENTIALS` environment variable set to the contents of your GSM service account key file.
+- [Poetry](https://python-poetry.org/docs/#installation) installed
+- [Pipx](https://pypa.github.io/pipx/installation/) installed
+
+### Running CAT in the same environment as our CI/CD pipeline (`airbyte-ci`)
 
-### Using `airbyte-ci`
 _Note: Install instructions for airbyte-ci are [here](https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connectors/pipelines/README.md) _
 
 **This runs connector acceptance and other tests that run in our CI**
 ```bash
 airbyte-ci connectors --name=<connector-name> test
 ```
 
-### Using Bash
+### Running CAT locally for Debugging/Development Purposes
 ```bash
-./acceptance-test-docker.sh
+# Hook up your GSM service account
+export GCP_GSM_CREDENTIALS=`cat <path-to-gsm-service-account-key-file>`
+
+# Install the credentials tool
+pipx install airbyte-ci/connectors/ci_credentials/
+
+# Writes the secrets to airbyte-integrations/connectors/source-faker/secrets
+VERSION=dev ci_credentials connectors/source-faker write-to-storage
```

Comment:
Also worth mentioning that this tool is cwd-sensitive. I wasted an hour figuring out why the secrets weren't being picked up when testing one connector: I had run this in the module root instead of the repo root and instead of mkdir-ing ./secrets it mkdir-ed ./connectors/source-postgres-strict-encrypt/secrets.

Alternatively, make this tool run correctly anywhere in the repo.